src/components/CachedOperations.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable react/require-default-props */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+// eslint-disable-next-line react/prefer-stateless-function
+class CachedOperations extends React.Component {
+  render() {
```

Comment:
Instead of disabling the rule, can we use a stateless function component instead?